src/components/Header/SearchBar/SearchBar.tsx

```diff
@@ -103,30 +105,39 @@ export const SearchBar = ({
           'mx-24 mt-24 md:m-0': displayLocation !== 'homepage',
         })}
       >
-        <input
-          type="text"
-          ref={textInput}
-          placeholder="Search"
-          className={cn(searchInput)}
-          value={query}
-          onChange={handleSearch}
-          style={{ ...extraInputStyle }}
-        />
-        <Icon name="icon-gui-search" size="24px" additionalCSS="absolute left-16 top-12" />
-        {!isInFocus && (
-          <div className="absolute right-16 top-12 hidden lg:flex items-center justify-end">
-            <KeyIcon className="mr-4">{keyIcon}</KeyIcon>
-            <KeyIcon className="pt-2">K</KeyIcon>
-          </div>
+        {externalScriptsData.inkeepEnabled ? (
+          <InkeepSearchBar
+            className={`${cn(searchInput)} ${cn(searchInputNoPadding)}`}
```

Comment:
Do we need more of this `cn` package? From what I can see it simply concatenates strings together, and the usages here only have one argument each so limited package usefulness aside, it wouldn't do anything

Reply:
I would advocate for removing all usages of this package where possible

Reply:
Was following convention within this file, have removed :+1: